lib/lookup.json

```
@@ -30,7 +30,7 @@
   },
   "ava": {
     "prefix": "v",
-    "skip": [true, "ppc", "win32", "x86", "rhel", "aix", "ia32", "10"],
+    "skip": ["ppc", "win32", "x86", "rhel", "aix", "ia32", "10"],
```

```suggestion
    "skip": ["ppc", "win32", "x86", "rhel", "aix", "ia32"],
```

`"10"` is Node.js 10